controller.js

```diff
@@ -0,0 +1,129 @@
+const testsList = document.getElementById("testsList");
+
+
+
+
+document.getElementById("createTestButton").querySelector("img").addEventListener("click", () => createTest() );
+document.getElementById("sendDataButton").addEventListener("click", function () {
+  sendData();
+});
+
+
+
+function sendData() {
+let xhr = new XMLHttpRequest();
+let url = "DataProccesor";
+xhr.open("POST", url, true);
```

Comment:
Maybe, `url` will be receiving value from some function in the future.
In that way, this change is not wrong.

Otherwise, you are right and this could be simplified.